### PR TITLE
Formatting changes for spawn list

### PR DESCRIPTION
**Pull request by JoanRisu**
Trying to figure out why Drake's itembox is not spawning with them. I am currently operating under the assumption that it is a formatting issue.

### CODE REVIEW

File: config/custom_items.txt [hunk 1]
```diff
@@ -183,19 +183,19 @@ item_path: /obj/item/weapon/storage/belt/champion
 }
 
 {
+ckey: drake_frostpaw
```
Comment (SpadesNeil) [inline config/custom_items.txt]: No underscores.

Comment (JoanRisu) [reply config/custom_items.txt]: Wait, how do I get this to him then?

Comment (SpadesNeil) [reply config/custom_items.txt]: drakefrostpaw

Comment (SpadesNeil) [reply config/custom_items.txt]: doof

**Pull request merged**